taxi/tests/test_model.py

```diff
@@ -0,0 +1,33 @@
+from django.test import TestCase
```

Comment:
between each group of imports add a blank line. Read recommendation here https://peps.python.org/pep-0008/#imports